modules/claims_api/app/models/claims_api/validated_token.rb

```diff
@@ -64,5 +30,17 @@ def client_credentials_token?
     def payload
       @validated_token_attributes
     end
+
+    def call_token_validation_service(token_validation_url, payload)
+      RestClient.post(token_validation_url,
+                      payload,
+                      { Authorization: "Bearer #{@token_string}",
+                        apiKey: Settings.claims_api.token_validation.api_key })
+    rescue => e
+      error = JSON.parse(e.response)
+      err_detail = error['errors'][0]['detail']
```

Comment:
This seems like it has null pointer potential, but I am not a rails expert.

Reply:
Will address. Better to be safe.

Reply:
Done.